hack/app_sre_create_image_catalog.sh

```diff
@@ -19,6 +19,9 @@ git clone \
     https://app:"${APP_SRE_BOT_PUSH_TOKEN}"@gitlab.cee.redhat.com/service/saas-osd-metrics-exporter-bundle.git \
     "$SAAS_OPERATOR_DIR"
 
+# ensure bundle directory exists when repository is new
+mkdir -p $"BUNDLE_DIR"
```

Comment:
I think this will make a directory called `BUNDLE_DIR`.
```suggestion
mkdir -p "$BUNDLE_DIR"
```